### PR TITLE
step-85: mathjax typo in Section L^2-Error

### PR DESCRIPTION
**Pull request by vyushut**

### CODE REVIEW

**Review by bangerth — APPROVED** [review]
I'm not sure that actually works -- we have some trouble because the text here is also copied into the table of contents at the top. The safer choice would probably be to write it with HTML markup: `<i>L<sup>2</sup></i>`. I'd accept that patch if you want to update your pull request!

That said, many thanks for tackling these little things!

File: examples/step-85/step-85.cc [hunk 1]
```diff
@@ -569,7 +569,7 @@ namespace Step85
 
 
 
-  // @sect3{$L^2$-Error}
+  // @sect3{\f$L^2\f$-Error}
```
Comment (tjhei) [inline examples/step-85/step-85.cc:572]: I would just do 
```suggestion
  // @sect3{L2-Error}
```

**Comment by tjhei** [comment]
Thank you, Vladimir. 👍

**Review by tjhei — APPROVED** [review]

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
Would you mind addressing the following error?
```
invalid author 'vyushut' without firstname and lastname

hint: for possible solutions, consult the webpage:
      https://github.com/dealii/dealii/wiki/Commit-authorship
```

**Comment by bangerth** [comment]
@vyushut I believe that the system will want you to change things retroactively for the previous commits as well. You might want to "squash" the three commits into one and use the username of the last commit.

**Comment by bangerth** [comment]
No, that's not making it any better -- now you have 7 commits for a one-line change :-) 

Maybe the easiest way to fix this is to close this PR and open another one that contains only one commit with the one-line change and with the user-name correctly set.

**Review by marcfehling — APPROVED** [review]

**Pull request merged**